lib/ansible/modules/system/filesystem.py

```diff
@@ -131,68 +129,68 @@ def main():
 
     # There is no "single command" to manipulate filesystems, so we map them all out and their options
     fs_cmd_map = {
-        'ext2' : {
-            'mkfs' : 'mkfs.ext2',
-            'grow' : 'resize2fs',
-            'grow_flag' : None,
-            'force_flag' : '-F',
+        'ext2': {
+            'mkfs': 'mkfs.ext2',
+            'grow': 'resize2fs',
+            'grow_flag': None,
+            'force_flag': '-F',
             'fsinfo': 'tune2fs',
         },
-        'ext3' : {
-            'mkfs' : 'mkfs.ext3',
-            'grow' : 'resize2fs',
-            'grow_flag' : None,
-            'force_flag' : '-F',
+        'ext3': {
+            'mkfs': 'mkfs.ext3',
+            'grow': 'resize2fs',
+            'grow_flag': None,
+            'force_flag': '-F',
             'fsinfo': 'tune2fs',
         },
-        'ext4' : {
-            'mkfs' : 'mkfs.ext4',
-            'grow' : 'resize2fs',
-            'grow_flag' : None,
-            'force_flag' : '-F',
+        'ext4': {
+            'mkfs': 'mkfs.ext4',
+            'grow': 'resize2fs',
+            'grow_flag': None,
+            'force_flag': '-F',
             'fsinfo': 'tune2fs',
         },
-        'reiserfs' : {
-            'mkfs' : 'mkfs.reiserfs',
-            'grow' : 'resize_reiserfs',
-            'grow_flag' : None,
-            'force_flag' : '-f',
+        'reiserfs': {
+            'mkfs': 'mkfs.reiserfs',
+            'grow': 'resize_reiserfs',
+            'grow_flag': None,
+            'force_flag': '-f',
             'fsinfo': 'reiserfstune',
         },
-        'ext4dev' : {
-            'mkfs' : 'mkfs.ext4',
-            'grow' : 'resize2fs',
-            'grow_flag' : None,
-            'force_flag' : '-F',
+        'ext4dev': {
+            'mkfs': 'mkfs.ext4',
+            'grow': 'resize2fs',
+            'grow_flag': None,
+            'force_flag': '-F',
             'fsinfo': 'tune2fs',
         },
-        'xfs' : {
-            'mkfs' : 'mkfs.xfs',
-            'grow' : 'xfs_growfs',
-            'grow_flag' : None,
-            'force_flag' : '-f',
+        'xfs': {
+            'mkfs': 'mkfs.xfs',
+            'grow': 'xfs_growfs',
+            'grow_flag': None,
+            'force_flag': '-f',
             'fsinfo': 'xfs_growfs',
         },
-        'btrfs' : {
-            'mkfs' : 'mkfs.btrfs',
-            'grow' : 'btrfs',
-            'grow_flag' : 'filesystem resize',
-            'force_flag' : '-f',
+        'btrfs': {
+            'mkfs': 'mkfs.btrfs',
+            'grow': 'btrfs',
```

Comment:
A bit related to my upper comment about moving the failure message for module not supporting resize of particular filesystem, it might make some sense to actually set the `grow` and/or `grow_flag` to `None`, and then essentially check for this value when user requests filesystem resizing to see if filesystem resizing is supported.

While at it, is it just me, or the 'grow_flag' is actually not used at all anywhere in the code? This is not related to your changes per-se, just something I tried to figure out quickly.